lua/xcodebuild/core/config.lua

```diff
@@ -124,6 +126,7 @@ local defaults = {
     oil_nvim = {
       enabled = true, -- enable updating Xcode project files when using oil.nvim
       guess_target = true, -- guess target for the file based on the file path
+      find_xcodeproj = false, -- instead of using configured xcodeproj search for xcodeproj closest to targeted file
```

Comment:
Please also update README.md - there is the same config there :).

Reply:
I just realized one more thing, but it would be more complex to implement. The current version won't handle properly if you are moving files between different projects. In other words, if you move your file from dir X where you have A.xcodeproj to dir Y where you have B.xcodeproj, then it should be first deleted from A.xcodeproj and then added to B.xcodeproj. Now it will be just moved within A.xcodeproj.

However, that would be much more complex for now, I think in the first version we can support only operations within the same xcodeproj.

Reply:
I see. I believe after refactoring it to have a single integration independent config we can find .xcodeproj for both X and Y, and if they don't match do delete and add rather than move